web/src/frontend/src/nav-soknad/redux/soknadsdata/soknadsdataContainerUtils.ts

```diff
@@ -42,7 +44,8 @@ export const connectSoknadsdataContainer = connect<{}, {}, SoknadsdataContainerP
 		lagreSoknadsdata,
 		oppdaterSoknadsdataSti,
 		setValideringsfeil,
-		settRestStatus
+		settRestStatus,
+		setVisSamtykkeInfo
```

Comment:
Fikk ikke ikke this.dispatch(setVisSamtykkeInfo(true)) til å fungere i infopanalet. Løsningen ble å legge inn action creatoren her i stedet, Ikke optimalt, men fungerer.